File: cardbattlemodule/src/main/java/com/syahputrareno975/cardbattlemodule/model/card/AllCardModel.java
package com.syahputrareno975.cardbattlemodule.model.card;

import java.io.Serializable;
import java.util.ArrayList;

public class AllCardModel implements Serializable {

    public ArrayList<CardModel> Cards = new ArrayList<CardModel>();

    public AllCardModel() {
    }

    public AllCardModel(ArrayList<CardModel> cards) {
        Cards = cards;
    }
}

File: app/src/main/java/com/syahputrareno975/simplecardbattle/model/card/AllCardModel.kt
package com.syahputrareno975.simplecardbattle.model.card

import com.syahputrareno975.simplecardbattle.model.player.PlayerModel
import java.io.Serializable

class AllCardModel :Serializable {

    var Cards = ArrayList<CardModel>()

    constructor()

    constructor(cards: ArrayList<CardModel>) {
        this.Cards = cards
    }

}